content/reference/sql/sql-support/miscellaneous.md

```diff
@@ -36,8 +36,8 @@ Dolt supports the following join hints:
 | MERGE_JOIN(<t1>,<t2>)    | ✅         | Use MERGE strategy joining two tables.                                                          |
 | HASH_JOIN(<t1>,<t2>)     | ✅         | Use HASH strategy joining two tables.                                                           |
 | INNER_JOIN(<t1>,<t2>)    | ✅         | Use INNER strategy joining two tables.                                                          |
-| SEMI_JOIN(<t1>,<t2>)     | ❌         | Use SEMI strategy joining two tables.                                                           |
```

Comment:
Why are these here?

Reply:
no idea. let me try add my summary change to the other pr